docs/connector-development/config-based/connector-builder-ui.md

```diff
@@ -0,0 +1,81 @@
+4. Build the connector's Docker image
+5. Use the built connector image in Airbyte
+
+### Run an Airbyte instance
+:::info
+You need at least Airbyte v0.40.19 to run the connector builder. 
+:::
+
+:::tip
+We recommend using a dedicated Airbyte instance for the connector builder UI. This will allow you to pick up connector builder UI upgrades without impacting your production data pipelines. 
+:::
+
+The connector builder UI is bundled as part of the Airbyte webapp. To run it, follow [the instructions for deploying Airbyte locally](deploying-airbyte/local-deployment) to run Airbyte on your machine. Once you do, Airbyte should be reachable at `http://localhost:8000`.
+
+
+### Visit the connector builder
+Once your Airbyte instance has started, visit `http://localhost:8000/`. You will be redirected to a URL of the form `localhost:8000/workspaces/<UUID>/connections` where `<UUID>` is the ID automatically generated for your workspace by Airbyte. 
+
+
+![The Airbyte webapp home page](./assets/lowcode_app_landing_screen.png)
+
+
+Visit `localhost:8000/workspaces/<UUID>/connector-builder` to find the connector builder UI. It should look like this: 
+
+![The Connector Builder home page](./assets/lowcode_landing_screen.png)
+
+You can now iterate on your connector's YAML manifest. See the [components](#components) section for more information on how to use the UI to iterate on the YAML manifest. 
+
+### Exporting the YAML
+Once you're done iterating on your YAML in the UI, you'll need to export it to your local filesystem into a connector module. 
+
+If you haven't already, create a low-code connector module using the connector generator (see [this YAML tutorial for an example](./tutorial/1-create-source.)) using the name you'd like to use for your connector. For this section, let's assume our connector is called `exchange-rates`. After creating the connector, overwrite the contents of `airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/exchange_rates.yaml` with the YAML you created in the UI. 
+
+
+<!-- TODO remove this when we bundle schema specification into the YAML -->
+:::info
+Remember that you need to declare the schemas of your streams separately from the YAML manifest. 
+
+You can do this by placing one `.json` file per stream in the `schemas/` directory of your connector module. Each `.json` should be a JSONSchema describing the schema of your stream. More information about this can be found in the [static schemas section of the Python CDK](../cdk-python/schemas#static-schemas). 
+:::
+
+### Building the connector image
+Follow the instructions in the connector README to build the Docker image. Typically this will be something like `docker build . -t airbyte/source-<name>:<version>`. 
+
+Once you've built the connector image, [follow these instructions](../../integrations/custom-connectors#adding-your-connectors-in-the-ui) to add your connector to your Airbyte instance. 
+
+## Connector Builder Components
```

Comment:
This section looks awesome! Nice job on documenting this 🙌

Reply:
thanks!